### PR TITLE
Additional config as variable for OLM Catalog Placement

### PR DESCRIPTION
**Pull request by deepsm007**
This is a temporary workaround

/cc @bear-redhat @openshift/test-platform

### CODE REVIEW

File: ci-operator/step-registry/hypershift/hostedcluster/create/hostedcluster/hypershift-hostedcluster-create-hostedcluster-ref.yaml [hunk 1]
```diff
@@ -38,6 +38,9 @@ ref:
   - name: HYPERSHIFT_INFRA_AVAILABILITY_POLICY
     default: "SingleReplica"
     documentation: "Availability policy for infrastructure services in guest cluster. Supported options: SingleReplica, HighlyAvailable, default SingleReplica"
+  - name: OLM_CATALOG_PLACEMENT
+    default: "guest"
+    documentation: "The OLM catalog placement."
```
Comment (Tal-or) [inline ci-operator/step-registry/hypershift/hostedcluster/create/hostedcluster/hypershift-hostedcluster-create-hostedcluster-ref.yaml:43]: Could you specify the other available options and explain about them?

Comment (deepsm007) [reply ci-operator/step-registry/hypershift/hostedcluster/create/hostedcluster/hypershift-hostedcluster-create-hostedcluster-ref.yaml]: added other options

**Comment by bear-redhat** [comment]
/pj-rehearse pull-ci-openshift-vertical-pod-autoscaler-operator-master-scorecard

**Comment by deepsm007** [comment]
/pj-rehearse pull-ci-openshift-vertical-pod-autoscaler-operator-master-scorecard

**Comment by deepsm007** [comment]
/pj-rehearse pull-ci-openshift-vertical-pod-autoscaler-operator-master-scorecard

**Pull request merged**